core/tests/protractor_desktop/navigation.js

```diff
@@ -139,6 +139,17 @@ describe('Meta Tags', function() {
   });
 });
 
+describe('DEV MODE Test', function() {
+  it('should not show Dev Mode label in prod', function() {
```

Comment:
Do we have a test for "Dev mode label" available in dev_mode? (CI only runs in prod mode.)

Reply:
I think the description should be changed. 
The test is for both prod and dev mode since the devmode value is gotten from constants